Demo/Demo/UIControls/ViewControllers/TabBarViewController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TabBarViewController.swift
+//  Demo
+//
+//  Created by Krunal Patel on 12/04/23.
+//
+
+import UIKit
+
+class TabBarViewController: UITabBarController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+    
+    private func setupUI() {
+        setupTabBar()
+    }
+    
+    private func setupTabBar() {
+        self.delegate = self
+        // Can't Create VC programmatically
```

Comment:
Why ? and you have used TableViewController instead of TabBarViewController

Reply:
Yes, i can, but got to know that later.